modules/sample/src/main/resources/petstore.json

```diff
@@ -514,7 +514,7 @@
           {
             "name": "custom-optional-value",
             "in": "formData",
-            "type": "number",
+            "type": "integer",
```

Comment:
Heh, I noticed the runtime warning about this, thought it was intentional ;)

Reply:
lol, no; it never happened in Scala because there was an `x-scala-type`